content/appendices.md

```diff
@@ -689,6 +692,8 @@ whereas more recent versions omit the domain and use a base64-encoded hash inste
 Event IDs are case-sensitive. They are not meant to be human-readable. They are
 intended to be treated as fully opaque strings by clients.
 
+The length of an event ID MUST NOT exceed 255 bytes.
+
```

Comment:
I feel like we should acknowledge that some room versions have much stricter requirements on event IDs which mean we can never get anywhere near 255 bytes:

```suggestion
In addition to the requirements of the room version, the length of an event ID,
including the `$` sigil and the domain where present, MUST NOT exceed 255 bytes.

Event IDs are case-sensitive. They are not meant to be human-readable. They are
intended to be treated as fully opaque strings by clients.
```